src/transformers/models/mbart/tokenization_mbart50_fast.py

```diff
@@ -0,0 +1,253 @@
+# coding=utf-8
+# Copyright 2021 The Facebook AI Research Team Authors and The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from contextlib import contextmanager
+from typing import List, Optional
+
+from tokenizers import processors
+
+from ...file_utils import is_sentencepiece_available
+from ...tokenization_utils import BatchEncoding
+from ...utils import logging
+from ..xlm_roberta.tokenization_xlm_roberta_fast import XLMRobertaTokenizerFast
+
+
+if is_sentencepiece_available():
+    from .tokenization_mbart50 import MBart50Tokenizer
+else:
+    MBart50Tokenizer = None
+
+
+logger = logging.get_logger(__name__)
+
+_all_mbart_models = ["facebook/mbart-50-large-one-to-many"]
+SPM_URL = "https://huggingface.co/facebook/mbart-50-large-one-to-many/resolve/main/sentencepiece.bpe.model"
+tokenizer_URL = "https://huggingface.co/facebook/mbart-50-large-one-to-many/resolve/main/tokenizer.json"
+
+FAIRSEQ_LANGUAGE_CODES = [
```

Comment:
Same remark as before, we can ignore the styler for this list and have it all on one line.